### ISSUE TITLE
Namespaces are useful

### ISSUE THREAD

**Issue by szepeviktor** [osDXP/osdxp-dashboard#42]
Please consider adding a backslash or use `use;` for non-namespaced function.

```php
use function add_filter;

// or

\add_filter();
```

**Comment by XedinUnknown** [comment]
Of course, the code works without that, because PHP will fall back to the global namespace when using functions.

But this would add more structure. I would say, `use` is better than FQN. More easily maintainable, readable, and in most cases results in less characters.

**Comment by szepeviktor** [comment]
Yes.

See https://github.com/slevomat/coding-standard#slevomatcodingstandardnamespacesfullyqualifiedglobalfunctions-
